src/index.js

```diff
@@ -1 +1,24 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import sendScores from './sendScore.js';
+import {
+  getScore, displayScores,
+} from './displayScore.js';
+
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/BT0dBrz8PnCOOMBT0tZ4/scores/';
+const form = document.getElementById('gameForm');
+const refreshBtn = document.getElementById('refreshBtn');
+const userName = document.getElementById('nameText');
+const userScore = document.getElementById('scoreText');
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  sendScores(url, userName, userScore);
+});
+
+refreshBtn.addEventListener('click', () => {
+  getScore(url).then((data) => { displayScores(data); });
+});
```

Comment:
thanks for the review. But I add the functionality of the refresh button and it is working. Here is the refrence

Reply:
- The refresh button is working fine. Thank you for making the changes requested by the previous reviewer. Great work! :dancers: